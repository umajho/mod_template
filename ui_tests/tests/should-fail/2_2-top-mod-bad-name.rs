fn main() {}

#[mod_template::define(define_foo)]
mod bad_name {}
