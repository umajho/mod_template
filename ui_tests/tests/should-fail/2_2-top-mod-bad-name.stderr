error: this should always be `__`, to emphasis that the attribute `mod_template::define` nullifies the name of the module it applied to
 --> tests/should-fail/2_2-top-mod-bad-name.rs:4:5
  |
4 | mod bad_name {}
  |     ^^^^^^^^
